docs/manual/cluster-admin/how-to-add-and-remove-nodes.md

```diff
@@ -6,19 +6,7 @@ OpenPAI doesn't support changing master nodes, thus, only the solution of adding
 
 ### Preparation
 
-To add worker nodes, please check if the nodes meet the following requirements:
-
-  - Ubuntu 16.04 (18.04 should work, but not fully tested.)
-  - Assign each node a **static IP address**, and make sure nodes can communicate with each other. 
-  - The nodes can access internet, especially need to have access to the docker hub registry service or its mirror. Deployment process will pull Docker images.
-  - SSH service is enabled and share the same username/password with current master/worker machines and have sudo privilege.
-  - (For CPU workers, you can ignore this requirement) **Have GPU and GPU driver is installed.**  You may use [a command](./installation-faqs-and-troubleshooting.md#how-to-check-whether-the-gpu-driver-is-installed) to check it. Refer to [the installation guidance](./installation-faqs-and-troubleshooting.md#how-to-install-gpu-driver) in FAQs if the driver is not successfully installed. If you are wondering which version of GPU driver you should use, please also refer to [FAQs](./installation-faqs-and-troubleshooting.md#which-version-of-nvidia-driver-should-i-install).
-  - **Docker is installed.**  You may use command `docker --version` to check it. Refer to [docker's installation guidance](https://docs.docker.com/engine/install/ubuntu/) if it is not successfully installed.
-  - (For CPU workers, you can ignore this requirement) **[nvidia-container-runtime](https://github.com/NVIDIA/nvidia-container-runtime) or other device runtime is installed. And be configured as the default runtime of docker. Please configure it in [docker-config-file](https://docs.docker.com/config/daemon/#configure-the-docker-daemon), because kubespray will overwrite systemd's env.**
-    - You may use command `sudo docker run nvidia/cuda:10.0-base nvidia-smi` to check it. This command should output information of available GPUs if it is setup properly.
-    - Refer to [the installation guidance](./installation-faqs-and-troubleshooting.md#how-to-install-nvidia-container-runtime) if the it is not successfully set up.
-  - OpenPAI reserves memory and CPU for service running, so make sure there are enough resource to run machine learning jobs. Check hardware requirements for details.
-  - Dedicated servers for OpenPAI. OpenPAI manages all CPU, memory and GPU resources of servers. If there is any other workload, it may cause unknown problem due to insufficient resource.
+To add worker nodes, please check if the nodes meet [the requirements for worker machines](./installation-guide.md#installation-requirements).
```

Comment:
It is not the same.

Installation requires workers should have GPU, which is not applied when add/remove node.

I suggest leaving this change to the bug bash period.